docs/api/expect.md

```diff
@@ -1405,3 +1405,70 @@ Don't forget to include the ambient declaration file in your `tsconfig.json`.
 :::tip
 If you want to know more, checkout [guide on extending matchers](/guide/extending-matchers).
 :::
+
+## expect.addEqualityTesters <Badge type="info">1.2.0+</Badge>
+
+- **Type:** `(tester: Array<Tester>) => void`
+
+You can use this method to define custom matchers to test if two object equals are equal. It is compatible with Jest's `expect.extend`.
```

Comment:
type?

```suggestion
You can use this method to define custom matchers to test if two objects are equal. It is compatible with Jest's `expect.addEqualityTesters`.
```

Reply:
Changed. I used `expect.extend` to indicate that user can define custom matchers and those matchers will work with custom testers as well.